20241014-wordcloud/src/routes/+page.svelte

```diff
@@ -0,0 +1,170 @@
+<script lang="ts">
+    import { onMount } from "svelte";
+    import WordCloud from "./WordCloud.svelte";
+
+    let words = [
+        {text: "placeholder", count: 1},
+    ];
+    let submission: string = "";
+    let submissionInput: HTMLInputElement;
+    const maxTimer = 5;
+
+    function invalidate(placeholder: string) {
+        submissionInput.placeholder = placeholder;
+        submissionInput.style.transition = "";
+        submissionInput.value = "";
+        submissionInput.style.backgroundColor = "#ffccdb";
+        setTimeout(() => {
+            submissionInput.style.transition = "background-color 0.5s";
+            submissionInput.style.backgroundColor = "white";
+        }, 100);
+        setTimeout(() => {
+            submissionInput.placeholder = "Enter a word...";
+        }, 600);
+    }
+
+    async function submitWord() {
+        const rgx = /drop\s+table/i;
```

Comment:
This is nice, but I bet there are more robust ways of protecting from SQL injection